packages/editor/src/lib/utils/sync/LocalIndexedDb.ts

```diff
@@ -303,6 +303,13 @@ export class LocalIndexedDb {
 			await assetsStore.put(blob, assetId)
 		})
 	}
+
+	async removeAsset(assetId: string) {
+		await this.tx('readwrite', [Table.Assets], async (tx) => {
+			const assetsStore = tx.objectStore(Table.Assets)
+			await assetsStore.delete(assetId)
+		})
+	}
```

Comment:
is it worth having this take the assetids array and do things in a single tx?

Reply:
maybe? I think i was worried that, if it's a lot of megabytes to delete it might start taking too long? I guess it's the same difference but maybe it could help the user mitigate freezing the ui potentially?

Reply:
I'm pretty skeptical that this would impact that without some sort of explicit collaborative scheduling but maybe?

Reply:
ok, fair enough, i'm sorta ambivalent on it. 👍